feature/metis/src/main/kotlin/de/tum/informatics/www1/artemis/native_app/feature/metis/ui/SinglePageConversationBodyViewModel.kt

```diff
@@ -0,0 +1,41 @@
+package de.tum.informatics.www1.artemis.native_app.feature.metis.ui
+
+import androidx.lifecycle.viewModelScope
+import de.tum.informatics.www1.artemis.native_app.core.common.flatMapLatest
+import de.tum.informatics.www1.artemis.native_app.core.data.retryOnInternet
+import de.tum.informatics.www1.artemis.native_app.core.datastore.authToken
+import kotlinx.coroutines.flow.SharingStarted
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.map
+import kotlinx.coroutines.flow.stateIn
+import androidx.lifecycle.ViewModel
+import de.tum.informatics.www1.artemis.native_app.core.data.service.network.AccountDataService
+import de.tum.informatics.www1.artemis.native_app.core.data.service.network.CourseService
+import de.tum.informatics.www1.artemis.native_app.core.datastore.AccountService
+import de.tum.informatics.www1.artemis.native_app.core.datastore.ServerConfigurationService
+import de.tum.informatics.www1.artemis.native_app.core.device.NetworkStatusProvider
+import de.tum.informatics.www1.artemis.native_app.core.model.account.isAtLeastTutorInCourse
+
+internal open class SinglePageConversationBodyViewModel(
+    courseId: Long,
+    accountService: AccountService,
+    serverConfigurationService: ServerConfigurationService,
+    courseService: CourseService,
+    accountDataService: AccountDataService,
+    networkStatusProvider: NetworkStatusProvider
```

Comment:
This viewmodel constructor is missing a coroutine context to match the signatures of the other view models. Please add the coroutine context.